src/core/Akka.MultiNodeTestRunner.Shared.Tests/TestRunShutdownSpec.cs

```diff
@@ -40,6 +41,8 @@ public void TestCoordinatorEnabledMessageSink_should_receive_TestRunTree_when_En
             foreach (var message in allMessages)
                 consoleMessageSink.Tell(message);
 
+            Task.Delay(1000).Wait();
```

Comment:
Technically, shouldn't need this - I was able to get the spec to pass without it on https://github.com/akkadotnet/akka.net/pull/2875